packages/components/src/range-control/index.js

```diff
@@ -89,10 +89,9 @@ function RangeControl( {
 				max={ max }
 				{ ...props } />
 			{ afterIcon && <Dashicon icon={ afterIcon } /> }
-			<input
+			<NumberControl
```

Comment:
Range control is a complex control in itself with validation logic that was recently added. The component already uses a BaseControl so using a NumberControl here would result in a BaseControl nested inside another BaseControl. I would prefer if we did not use NumberControl inside RangeControl at least for now.